.gitattributes

```diff
@@ -10,5 +10,12 @@
 *.xml text eol=lf
 *.dist text eol=lf
 *.neon text eol=lf
```

Comment:
Please set this to `export-ignore` too.

Reply:
I don't like very much export-ignoring asterisks, there is no control on what in the future could be ignored.
phpstan.neon was already added few line below.

Reply:
Then remove this line please. phpstan.neon is the only *.neon file at the moment.